src/OpenLoco/Audio/Audio.cpp

```diff
@@ -875,13 +878,117 @@ namespace OpenLoco::Audio
+            size_t wildernessCount = 0; // bh
+            size_t treeCount = 0;       //cx
+            for (auto& tilePos : searchRange)
+            {
+                const auto tile = Map::TileManager::get(tilePos);
+                bool passedSurface = false;
+                for (const auto& el : tile)
+                {
+                    auto* elSurface = el.as<Map::SurfaceElement>();
+                    if (elSurface != nullptr)
+                    {
+                        passedSurface = true;
+                        if (elSurface->water() != 0)
+                        {
+                            waterCount++;
+                            break;
+                        }
+                        else if (elSurface->var_4_E0() && elSurface->isLast())
+                        {
+                            wildernessCount++;
+                            break;
+                        }
+                        else if (elSurface->baseZ() >= 64 && elSurface->isLast())
+                        {
+                            wildernessCount++;
+                            break;
+                        }
+                        continue;
+                    }
+                    auto* elTree = el.as<Map::TreeElement>();
+                    if (passedSurface && elTree != nullptr)
+                    {
+                        const auto* treeObj = ObjectManager::get<TreeObject>(elTree->treeObjectId());
+                        if (!(treeObj->flags & TreeObjectFlags::droughtResistant))
+                        {
+                            treeCount++;
+                        }
+                    }
+                }
+            }
+
+            if (waterCount > 60)
+            {
+                ambientSound = PathId::css3;
+            }
+            else if (wildernessCount > 60)
+            {
+                ambientSound = PathId::css2;
+            }
+            else if (treeCount > 30)
+            {
+                ambientSound = PathId::css4;
+            }
+        }
+        auto* channel = getChannel(ChannelId::ambient);
+        if (channel == nullptr)
+        {
+            return;
+        }
+        // In these situations quieten until channel stopped
+        if (!ambientSound.has_value() || (channel->isPlaying() && _chosenAmbientNoisePathId != *ambientSound))
+        {
+            const auto newVolume = channel->getAttributes().volume - 100;
+            if (newVolume < -3500)
+            {
+                stopChannel(ChannelId::ambient);
+            }
+            else
+            {
+                channel->setVolume(newVolume);
+            }
+            return;
+        }
+
+        if (_chosenAmbientNoisePathId != *ambientSound)
+        {
+            auto path = Environment::getPath(*ambientSound);
+            if (loadChannel(ChannelId::ambient, path, 0))
+            {
+                playChannel(ChannelId::ambient, true, -3500, 0, 0);
```

Comment:
`maxVolume`

Reply:
again this is a minVolume constant for initialy starting the sound. It will slowly ramp up from -3500 to maxVolume if you stay over the same area.